pydash/arrays.py

```diff
@@ -1443,6 +1487,58 @@ def union(*arrays):
     return uniq(flatten(arrays))
 
 
+def union_by(*arrays, **kargs):
```

Comment:
Take a look at the intersection() functions. We want to do the same function signature for the union() functions.